CHANGELOG.md

```diff
@@ -38,6 +38,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+### State Machine Breaking
+- (feemarket) [tharsis#1105](https://github.com/tharsis/ethermint/pull/1105) BaseFee calculation based on GasWanted instead of GasUsed.
```

Comment:
```suggestion
### State Machine Breaking

- (feemarket) [tharsis#1105](https://github.com/tharsis/ethermint/pull/1105) Update `BaseFee` calculation based on `GasWanted` instead of `GasUsed`.
```